internal/runbits/requirements/dependencies.go

```diff
@@ -0,0 +1,217 @@
+package requirements
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+
+	"github.com/go-openapi/strfmt"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+	"github.com/ActiveState/cli/pkg/platform/runtime/buildplan"
+	"github.com/ActiveState/cli/pkg/platform/runtime/store"
+	"github.com/ActiveState/cli/pkg/platform/runtime/target"
+)
+
+// maxListLength is the maximum number of direct dependencies to show before adding a "more..."
+// sentinel.
+const maxListLength = 10
```

Comment:
Playing with this PR and seeing the output, I'm going back and forth on whether to limit the list. The issue I see is that even with `state info` you would not be able to retroactively see what dependencies were brought in with a `state install` that you just ran. 

I'm not able to come up with a solution right here on the spot that doesn't have caveats. Let's discuss during stand how we might want to handle this.

Reply:
Based on discussion during stand, I've removed the list truncation. All dependencies are shown.